commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -91,6 +127,38 @@ public static CommerceOrder addB2CCommerceOrder(
 			commerceAccount.getCommerceAccountId(), commerceCurrencyId);
 	}
 
+	public static CommerceOrder addB2CCommerceOrder(
+			long groupId, long userId, long commerceCurrencyId,
```

Comment:
@ethanbustad @riccardo-alberti   for what is used the groupId parameter?

Reply:
It is used to get the service context that's used to create the commerce account